### PR TITLE
complated oop-exersice

### PR DESCRIPTION
**Pull request by maxim-zabolotny**
### oop-exersice

[Demo](https://maxim-zabolotny.github.io/tiny-js-world.github.io/)
[Code base](https://github.com/maxim-zabolotny/tiny-js-world.github.io)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/maxim-zabolotny/oop-exersice/index.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    }
+  
+    toString() {
+      let template = [
+        `name: ${this.name}; `,
+        `gender: ${this.gender}; `,
+        `species: ${this.species}; `,
+        `saying: ${this.saying}; `,
+        `legs: ${this.legs}; `,
+      ];
```
Comment (OleksiyRudenko) [inline submissions/maxim-zabolotny/oop-exersice/index.js:23]: Does it really contain template?
Each fragment here ends with `; `
Why not to use this repeating delimiter in `join` below?

File: submissions/maxim-zabolotny/oop-exersice/index.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  const man = new Man('Elon');
+  man.addFriends(cat, woman);
+  woman.addFriends(man, dog);
+  dog.addFriends(cat, woman, man);
+  
+  const creatures = [dog, cat, woman, man];
+  
+  creatures.forEach((creature) => {
+    print(creature);
+  });
```
Comment (OleksiyRudenko) [inline submissions/maxim-zabolotny/oop-exersice/index.js:79]: Make sure your files end with a newline characters.
Let your IDE do this dirty job for you.

**Review by OleksiyRudenko — CHANGES_REQUESTED** [review]
@maxim-zabolotny good job!
A few improvements and we're done.

File: submissions/maxim-zabolotny/oop-exersice/index.js (hunk 2 above)
Comment (maxim-zabolotny) [reply submissions/maxim-zabolotny/oop-exersice/index.js]: I hope I understood your comments correctly.
Please check my code again

Comment (maxim-zabolotny) [reply submissions/maxim-zabolotny/oop-exersice/index.js]: I added an empty line at the end of the file, checked its presence in two editors, but it is not visible on the github.

**Comment by OleksiyRudenko** [comment]
@maxim-zabolotny 

(1) This PR now contains a couple of strange files.

(2) Also your original app code base and, consequently, demo are not updated.

If you tested it you might have notice that there is yet another `join` to change.

The requirements in the instructions are for good reasons. Please take care of mentors' and use their capacity frugally.

image

File: submissions/maxim-zabolotny/oop-exersice/index.js (hunk 2 above)
Comment (maxim-zabolotny) [reply submissions/maxim-zabolotny/oop-exersice/index.js]: Deleted unnecessary files. Corrected your comments.
Updated demo application

**Review by OleksiyRudenko — APPROVED** [review]
@maxim-zabolotny well done!

**Pull request merged**